README.md

```
@@ -103,8 +103,23 @@ $ vtr test -- option1 option2
 # This will run the task "test" with the arguments "option1" and "option2"
 ```
 
-If your task uses an `${input:id}` variable, you can provide the value for
-this variable as an environment variable named `VTR_INPUT_{id}`. Example:
+If your task uses an `${input:id}` variable, you can provide the value in three ways
+(in order of precedence):
+
+1. **CLI flags** (recommended, cross-platform):
+   ```bash
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)